shopify/product-reviews-template/api/models/shopifyShop/actions/install.js

```diff
@@ -0,0 +1,137 @@
+import {
+  applyParams,
+  save,
+  ActionOptions,
+  InstallShopifyShopActionContext,
+} from "gadget-server";
+
+/**
+ * @param { InstallShopifyShopActionContext } context
+ */
+export async function run({ params, record, logger, api, connections }) {
+  applyParams(params, record);
+
+  const shopify = connections.shopify.current;
+
+  const reviewMetaobjectDefinitionCreateResponse = await shopify.graphql(
+    `mutation ($definition: MetaobjectDefinitionCreateInput!) {
+      metaobjectDefinitionCreate(definition: $definition) {
+        metaobjectDefinition {
+          id
+        }
+        userErrors {
+          message
+        }
+      }
+    }`,
+    {
+      definition: {
+        type: "review",
```

Comment:
The installation will fail if any store already has a "review" type.
I think we want to make it an app-owned metaobject like Shopify's example: https://shopify.dev/docs/apps/build/custom-data#metaobject-example